src/views/OrderPage/index.ts

```diff
@@ -0,0 +1 @@
+export * from '../OrderPage/OrderPage.styles';
```

Comment:
Bad export, you are exporting OrderPage styles instead of the component. This is breaking the app, it can't be run. You are also going outside of the folder the index is currently in and going back into it. I believe you mean to add the `index.ts` to the parent folder instead of this one.